### ISSUE TITLE
less displays blank powershell window with no visible text

### ISSUE THREAD

**Issue by djduncan585** [Pscx/Pscx#88]
Powershell window is using default colors. When using less to read a text file (less myfile.txt) or when piping a file to it (cat myfile.txt | less), no text is visible. Highlighting text with the mouse displays nothing. less can be exited as normal using :q, but no typed text is displayed either. However, when reading the file into less using the less myfile.txt syntax, the path to the file is displayed at the bottom of the screen initially.

**Comment by rkeithhill** [comment]
I've run into this problem as well.  What version of PSCX and what version of less (`less --version`) do you have.  Also, are you running PowerShell v5, 6 or 7 in the native console or in Windows Terminal?

**Comment by rkeithhill** [comment]
FYI, I'm using 553x and I don't have the non-visible text issue.  You can install that version of less via chocolatey `choco install less`.

**Comment by djduncan585** [comment]
Looks like I was using the default version 5 that comes with Windows 10 in the native console. I was able to initially work around this problem by upgrading to version 7 and installing Windows Terminal, which does not exhibit the symptoms that version 5 does. Version of less was 394; I just upgraded to 590 using Chocolatey, and the symptoms are no longer manifesting in the instance of PS version 5 in the native console, and they are also not manifesting when PS version 5 is running in a VS Code terminal window. 
Thank you.
